syz-cluster/series-tracker/main.go

```diff
@@ -170,6 +183,32 @@ func (sf *SeriesFetcher) handleSeries(ctx context.Context, series *lore.Series,
 	return nil
 }
 
+type seriesProcessor struct {
+	emails map[string]struct{}
+}
+
+var errFailedToParse = errors.New("failed to parse the email")
+
+func (sp *seriesProcessor) Process(raw []byte) ([]byte, error) {
+	msg, err := email.Parse(bytes.NewReader(raw), nil, nil, nil)
+	if err != nil {
+		return raw, fmt.Errorf("%w: %w", errFailedToParse, err)
+	}
+	if sp.emails == nil {
+		sp.emails = map[string]struct{}{}
+	}
+	for _, email := range msg.Cc {
+		sp.emails[email] = struct{}{}
+	}
+	return []byte(msg.Body), nil
+}
+
+func (sp *seriesProcessor) Emails() []string {
+	list := slices.Collect(maps.Keys(sp.emails))
```

Comment:
"golang.org/x/exp/maps" provides maps.Keys() that returns the slice.
"maps" provides maps.Keys() that returns the generator.
slices.Collect() is needed to convert this generator into the slice.

"golang.org/x/exp"  description is `This subrepository holds experimental and deprecated (in the old directory) packages.`

We're already using `golang.org/x/exp/maps` in a few places.

It looks Ok to me to use "maps" or "golang.org/x/exp/maps" in the single file.
Alternatively we can agree to use only the standard "maps".
CC @dvyukov

Reply:
I think it might be an overkill to have to deal with `This subrepository holds experimental and deprecated (in the old directory) packages.` only to replace `slices.Collect(maps.Keys(sp.emails))` with `sp.emails.Keys()`.